pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "aladdin"
-version = "1.19.7.3"
+version = "1.19.7.4"
```

Comment:
wow! we're already on x.4?? it's only been a week ;)

Reply:
We need to update the aladdin/__init__.py file too right?

Reply:
haha 😉 I updated the `init.py` file so it does not need to be updated anymore! it uses `importlib` to figure out the package version instead 🙂